packages/carbon-graphs/src/js/controls/Gantt/helpers/eventHelpers.js

```diff
@@ -135,10 +135,9 @@ const loadEventInput = (inputJSON) => {
  * @param {object} scale - d3 scale for Graph
  * @param {object} gantt - Graph config object for the content.
  * @param {object} trackGroupPath - Container for the track
- * @param {object} legendSVG - d3 element path of the legend from the parent control
  * @returns {undefined} - returns nothing
  */
-const reflowEvents = (config, scale, gantt, trackGroupPath, legendSVG) => {
+const reflowEvents = (config, scale, gantt, trackGroupPath) => {
```

Comment:
looks like you removed the parameter but the argument is still being passed
https://github.com/cerner/terra-graphs/blob/remove-use-of-document-in-reflow/packages/carbon-graphs/src/js/controls/Gantt/Track.js#L345

Reply:
removed it here https://github.com/cerner/terra-graphs/pull/18/commits/30019d1a1afbecf20ff1b8476b29e225df1cc68b